pinot-query-runtime/src/test/java/org/apache/pinot/query/mailbox/GrpcMailboxServiceTest.java

```diff
@@ -78,7 +78,8 @@ public void testHappyPath()
     JsonMailboxIdentifier mailboxId = new JsonMailboxIdentifier(
         "happypath",
         new VirtualServerAddress("localhost", _mailboxService1.getMailboxPort(), 0),
-        new VirtualServerAddress("localhost", _mailboxService2.getMailboxPort(), 0));
+        new VirtualServerAddress("localhost", _mailboxService2.getMailboxPort(), 0),
```

Comment:
self-review: Use static constants for sender/receiver stage-id here and in other places.